### ISSUE TITLE
Demo_single_shot_video.ipynb  question

### ISSUE THREAD

**Issue by fsfing** [Waller-Lab/UDN#1]
@monakhova
hello，monakhova：
I have encountered a problem when reading your code, and I would like to ask you for advice.

In Demo_single_shot_video.ipynb, I have two questions.
question1:
    Does this demo file realize your thesis Fig. 5 or e Visualization 1?
question2:
simulated = True

in  "loss = mse(gen_meas, meas_ts)". The internal data of gen_meas is a complex number, and the internal data of meas_ts is a real number. Directly performing MSE operations on them will cause an error.
